### PR TITLE
cephfs: Add volumesnapshotclass for external-storage

### PR DESCRIPTION
**Pull request by Madhu-1**
Add volumesnapshotclass yamls to  more tests for cephfs external-storage later

update #2015

### CODE REVIEW

**Review by iPraveenParihar — DISMISSED** [review]

File: scripts/k8s-storage/volumesnapshotclass-cephfs.yaml.in [hunk 1]
```diff
@@ -0,0 +1,11 @@
+---
+apiVersion: snapshot.storage.k8s.io/v1
+kind: VolumeSnapshotClass
+metadata:
+  name: k8s-storage-e2e-cephfs
+driver: cephfs.csi.ceph.com
+parameters:
+  clusterID: @@CLUSTER_ID@@
+  csi.storage.k8s.io/snapshotter-secret-name: csi-cephfs-secret
+  csi.storage.k8s.io/snapshotter-secret-namespace: default
```
Comment (nixpanic) [inline scripts/k8s-storage/volumesnapshotclass-cephfs.yaml.in]: should the namespace not be `rook-ceph`?

**Comment by nixpanic** [comment]
This needs backporting because CI jobs will fail after merging #4542 and the script does not exist.

File: scripts/k8s-storage/volumesnapshotclass-cephfs.yaml.in (hunk 1 above)
Comment (Madhu-1) [reply scripts/k8s-storage/volumesnapshotclass-cephfs.yaml.in]: we use rook created secrets, i was also confused but after thinking a lot and checking other places we are good 👍🏻

Comment (nixpanic) [reply scripts/k8s-storage/volumesnapshotclass-cephfs.yaml.in]: Can you explain why `sc-rbd.yaml.in` uses `rook-ceph` for all namespaces?

```yaml
csi.storage.k8s.io/provisioner-secret-namespace: rook-ceph
```

Comment (nixpanic) [reply scripts/k8s-storage/volumesnapshotclass-cephfs.yaml.in]: The `csi-cephfs-secret` is also not created by Rook, I guess?

Comment (Madhu-1) [reply scripts/k8s-storage/volumesnapshotclass-cephfs.yaml.in]: mistake, i though we are talking about why we are using `rook-ceph` which i was added initially but messed up somehow, updated to use rook created secret

Comment (Madhu-1) [reply scripts/k8s-storage/volumesnapshotclass-cephfs.yaml.in]: @nixpanic updated now PTAL

**Review by iPraveenParihar — APPROVED** [review]

**Review by nixpanic — APPROVED** [review]

File: scripts/k8s-storage/volumesnapshotclass-cephfs.yaml.in (hunk 1 above)
Comment (nixpanic) [reply scripts/k8s-storage/volumesnapshotclass-cephfs.yaml.in]: Thanks

**Comment by nixpanic** [comment]
@mergifyio rebase

**Comment by nixpanic** [comment]
@mergifyio queue

**Comment by Madhu-1** [comment]
@Mergifyio refresh

**Pull request merged**